deploy/helm/scf/values.yaml

```diff
@@ -1,2 +1,6 @@
 system_domain: ~
 deployment_name: scf
+releases:
+  # The defaults for all release, where we do not otherwise override them.
```

Comment:
release -> releases

Reply:
Fixed in d99eded3f0ac6b69429ae70215b5f93052b910aa (which I force-pushed because I forgot, sorry)